modules/local/resolve_found_neoepitopes.nf

```diff
@@ -36,7 +24,7 @@ process RESOLVE_FOUND_NEOEPITOPES {
             -o ${prefix}
 
         cat <<-END_VERSIONS > versions.yml
-        ${getProcessName(task.process)}:
+        ${task.process}:
```

Comment:
Actually, this seems to be typically 
```suggestion
        "${task.process}":
```
But I am not sure if this has any impact.